### DISCUSSION TITLE
Video Cards

### DISCUSSION THREAD

**Discussion by wdmasters** [THUDM/CogVideo#631] [Q&A] [answered]
Will a Intel Arc Video Card with VRAM 16GB work with your App?
Thanks

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
Cannot, the memory usage mentioned in README has been optimized, and these optimizations and tools need to depend on CUDA (used in the implementation of the diffusers library). Therefore, the video memory required to run on non-CUDA devices is far more than this number, and an ARC graphics card with 16G of video memory is unable to complete the task.
